docker/nginx/nginx.conf

```diff
@@ -0,0 +1,105 @@
+# /etc/nginx/nginx.conf
```

Comment:
small nit. 
Does this file differ from the default one that is created in `/etc/nginx/nginx.conf` when we run `apk add nginx`? If not, can we continue to rely on that one?

Reply:
I didn't try, I'll try reverting the change

Reply:
I've tried to update this to the Netflix lemur and they also seem to have left it to rot. The default config doesn't seem to work, so I'll keep this change